src/gratia_cleanup.py

```diff
@@ -0,0 +1,17 @@
+#!/bin/env python
+
+import os
+import datetime
+
+os.environ['CONDOR_CONFIG'] = '/etc/condor-ce/condor_config'
+
+import htcondor
+
+GRATIA_DIR = htcondor.param['PER_JOB_HISTORY_DIR']
+HISTORY_FILES = [os.path.join(GRATIA_DIR, x) for x in os.listdir(GRATIA_DIR)]
```

Comment:
I'm a bit worried about listing an entire directory, if it gets out of hand.  What if there are lots, and lots of files in this directory.